app/assets/javascripts/modules/CaseClosure.js

```diff
@@ -25,6 +25,9 @@ moj.Modules.CaseClosure = {
   $sarIrOutcomeOverturned : $('#sar_internal_review_sar_ir_outcome_overturned'),
   $outcomeExplanationGroups : $('.responsible-for-outcome-group, .outcome-reasons-group'),
 
+  $otherOverturned : $('.js-other-overturned'),
+  $outcomeReasonOtherOption : $('label:contains("Other")').siblings().first(),
```

Comment:
Would it be more robust to put an ID on the bit your targeting here? Or is it that you want it to break if any HTML get's stuck in between, since you have a test for it.  Or maybe it's just more headache. Just question really.